app.py

```
@@ -171,11 +173,271 @@ def render_settings(action):
+        logging.debug(f"Server: {server}")
+        logging.debug(f"Database: {database}")
+        logging.debug(f"Table: {table}")
+        logging.debug(f"Row Index: {row_index}")
+        logging.debug(f"Column Names: {column_names}")
+        logging.debug(f"Row Data: {row_data}")
+        logging.debug("=" * 80)
+
+        result = mdb.update_row(db, server, database, table, row_index, column_names, row_data)
+        logging.debug(f"Update result: {result}")
+        return jsonify(result)
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)